LICENSE.md

```diff
@@ -0,0 +1,504 @@
+		  GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
Sorry for the confusion, I don't think we need this license file but instead need the corresponding `.licenseheader` file. This way (once it is supported in rider) the licenseheaders are getting inserted automatically into files that don't have them.

You used the right file as template for the `.cs` files. This file should be placed in the root and projects should link to it. You can take a look at how this is done in the re-motion projects.